app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -20,12 +20,9 @@ import kotlinx.android.synthetic.main.main_fragment.*
 
 class MainFragment : Fragment() {
 
-    val CAMERA_PERMISSION_REQUEST_CODE = 1999
-    val CAMERA_REQUEST_CODE = 1998
+    val CAMERA_PERMISSION_REQUEST_CODE: Int = 1999
```

Comment:
This wasn't a huge one, but I noticed that the variable have an access modifier of private, just in case you plan on using this file more, having it as private can cause a lot less confusion. Also if you look at the naming conventions using it create variables you can see that they use "Variablename: (type) = value)" see more here - https://kotlinlang.org/docs/reference/coding-conventions.html